runtime/compiler/optimizer/J9EstimateCodeSize.hpp

```diff
@@ -77,6 +78,7 @@ class TR_J9EstimateCodeSize : public TR_EstimateCodeSize
 
    protected:
       bool estimateCodeSize(TR_CallTarget *, TR_CallStack * , bool recurseDown = true);
+      TR::CFG &generateCFG(TR_CallTarget *calltarget, TR::Region &cfgRegion, TR_J9ByteCodeIterator &bci, NeedsPeekingHeuristic *nph, TR::Block** blocks, flags8_t * flags, int &size, bool& thisOnStack, bool& hasThisCalls, bool& unresolvedSymbolsAreCold);
```

Comment:
Could you double-check if some arguments are really necessary? `int &size` for example shouldn't need to be an argument. it's currently initialized once in `realEstimateCodeSize` and used only inside `generateCFG `.  I think `hasThisCalls` and `thisOnStack` can be removed from the argument as well. The read/writes of those 2 variables outside of `generateCFG ` are bogus I think.

Reply:
So size is reference only on a debug message. But the value of size can still be obtained without passing a reference because it is stored on `calltarget->_fullSize`. So, after the `processBytecodeAndGenerateCFG` call I just initialize a new `size` variable to the contents of `calltarget->_fullSize`.